rspace/src/test/scala/coop/rchain/rspace/StorageActionsTests.scala

```diff
@@ -213,6 +213,35 @@ trait StorageActionsTests[F[_]]
       _             = insertActions should have size 0
     } yield ()
   }
+  "consuming and then producing on the same channel with persistent flag" should
+    "return the continuation and data and remove the persistent data" in fixture {
```

Comment:
Shouldn't this say

>  "return the continuation and data and **not insert** the persistent data"

This gave me chills :joy:

Reply:
You're right!